dbms/src/Databases/DatabaseOrdinary.cpp

```diff
@@ -180,7 +180,18 @@ void DatabaseOrdinary::loadStoredObjects(
     auto & external_loader = context.getExternalDictionariesLoader();
     external_loader.addConfigRepository(getDatabaseName(), std::move(dictionaries_repository));
     bool lazy_load = context.getConfigRef().getBool("dictionaries_lazy_load", true);
-    external_loader.reload(!lazy_load);
+
+    auto filter = [this](const std::string & dictionary_name) -> bool
+    {
+        if (!startsWith(dictionary_name, name + "." /* db name */))
+        {
+            LOG_DEBUG(log, "Ignore dictionary " << dictionary_name << ", in database " << name);
```

Comment:
Is this message clear enough for the user? Wouldn't the user wonder why dictionary loading was ignored?

Reply:
It has `in database ...` to avoid possible confusions, but can be removed